package.json

```diff
@@ -39,7 +39,7 @@
     "stylelint": "^7.13.0",
     "stylelint-config-standard": "^16.0.0",
     "url-loader": "^0.5.7",
-    "webpack": "^3.0.0",
+    "webpack": "^3.4.1",
```

Comment:
`extract-text-webpack` 3.0 is not compatible with older webpack versions (https://github.com/webpack-contrib/extract-text-webpack-plugin/issues/581). I found 3.4.1 is able to compile without any other changes.